default/web_tt2/config_common.tt2

```diff
@@ -1,262 +1,269 @@
+<!-- config_common.tt2 -->
```

Comment:
As this file is included by the other files, HTML comments might break markup.  IMHO following things may be better.
```
[%# config_common.tt2 ~%]
...
[%# end config_common.tt2 ~%]
```

Reply:
Fixed in https://github.com/sympa-community/sympa/pull/582/commits/7b3a39c6ac7c013a0990a7033f81118d1943e2de

Reply:
Confirmed.